app/models/invoice/data_presenter.rb

```diff
@@ -49,9 +50,9 @@ def order_completed_at
     end
 
     def checkout_adjustments(exclude: [], reject_zero_amount: true)
-      adjustments = all_eligible_adjustments
+      adjustments = all_eligible_adjustments.map(&:clone)
 
-      adjustments.reject! { |a| a.originator_type == 'Spree::TaxRate' }
+      adjustments.reject! { |a| a.originator.type == 'Spree::TaxRate' }
```

Comment:
I think this was right before, it should be `a.originator_type`...

Reply:
the `originators` connected to the `adjustments` are serialized now.
because in some cases, I need to access the originator's amount (when the originator is a tax rate).